LSPS6/README.md

```diff
@@ -0,0 +1,89 @@
+# LSPS6 LSP meta-data
+
+| Name    | `meta-data` |
+| ------- |---------------|
+| Version | 1             |
+| Status  | Draft         |
+
+LSPS6 requires [BOLT8][] as a transport layer.
+
+[BOLT8]: https://github.com/lightning/bolts/blob/master/08-transport.md
+
+## Motivation
+
+Sometimes the client wants to obtain extra information from the LSP.
+There might be information that is not available through regular LSPS messages.
+An example for such information could be the pricing strategy for LSPS1.
+This extra information can be in the form of a URL or an email address.
+We need a way to prevent spoofing.
+
+### Actors
+
+The 'LSP' is the API provider, and acts as the server.
+The 'client' is the API consumer.
+
+## DNS Public Key Verification
+
+[BIP353][] provides a way to discover payment instructions over DNS.
+Analogous to this the LSP SHOULD create a DNS TXT record in the form of
+`lsps.acme.com. 3600 IN TXT "public_key:0326e6...2833a3"` for public key verification.
+
+[BIP353]: https://github.com/bitcoin/bips/blob/master/bip-0353.mediawiki
+
+## API
+
+### lsps6.get_info
+
+`lsps6.get_info` provides generic information about the LSP like icon, logo, etc.
+
+It is possible to confirm authority over DNS when there are URLs (or an email address) in the response.
```

Comment:
```suggestion
A client MAY confirm the authority of this node public key over DNS when there are URLs (or an email address) in the response.
```

- Is authority the correct word? Isn't it `authenticity`?
- Does the client need to verify all URLS? Shouldn't one be enough?